### PR TITLE
Optimize simulation settings and fix gradient bug

### PR DESCRIPTION
**Pull request by proteneer**
Addresses #486

### CODE REVIEW

File: fe/estimator_abfe.py [hunk 1]
```diff
@@ -132,6 +132,10 @@ def simulate(lamb, box, x0, v0, final_potentials, integrator, barostat, equil_st
     equil_schedule = np.ones(equil_steps)*lamb
     ctxt.multiple_steps(equil_schedule)
 
+    # (ytz): intentionally hard-coded, I'd rather the end-user *not*
```
Comment (proteneer) [inline fe/estimator_abfe.py:135]: (note that equilibration will still use whatever default barostat interval is used, but it's assumed that once we're at equilibrium, interval=25 should be sufficient).

**Review by maxentile — APPROVED** [review]
Looks good. One point you mentioned in linked issue https://github.com/proteneer/timemachine/issues/486 is to change the du/dp save interval -- should that be done in the current PR as well?

**Comment by proteneer** [comment]
> Looks good. One point you mentioned in linked issue #486 is to change the du/dp save interval -- should that be done in the current PR as well?

It is done here in this PR:
```
        du_dp_obs.append(custom_ops.AvgPartialUPartialParam(impl, 25))
```

**Comment by maxentile** [comment]
Ahh! Thanks, I had just looked for an interval of "100" and didn't see it. Looks like it addresses all 3 points then.

**Comment by proteneer** [comment]
yeah, I re-ran with an interval of 25 and it didn't slow things down much compared to 100, so figured might as well as collect some more statistics!

**Pull request merged**